changelog/std-traits-hasStaticMember.dd

```diff
@@ -0,0 +1,13 @@
+Added `std.traits.hasStaticMember` to check whether a symbol is a static member of a type.
```

Comment:
@andralex while the current changelog parser is very forgiving, we should probably make a decision on the format:
- (A) title (can be multiple lines) - empty line - description
- (B) title (one line) - empty line (optional) - description

(A) is implemented as https://github.com/dlang/tools/pull/220
(B) is how our parser works currently.
Personally I am quite happy with closing the PR and agreeing that a title can't be more than one line (doesn't make much sense anyways).